sdk/parents/azure-arm-parent/pom.xml

```diff
@@ -672,7 +672,7 @@
         <configuration>
           <doctitle>Microsoft Azure Management Libraries for Java ${project.version}</doctitle>
           <windowtitle>Microsoft Azure Management Libraries for Java ${project.version}</windowtitle>
-          <additionalparam>-Xdoclint:none</additionalparam>
+          <doclint>false</doclint>
```

Comment:
`<additionalparam>-Xdoclint:none</additionalparam>`	and `<doclint>false</doclint>` are essentially the same right?

Reply:
Yes and `additionalparam` isn't a valid configuration option for the Javadoc plugin. https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html#doclint